tests/trace/test_weaveflow.py

```diff
@@ -119,46 +118,6 @@ def op_with_unknown_param() -> int:
     assert op_with_unknown_param() == 12
 
 
-@pytest.mark.skip("artifact file download doesn't work here?")
-def test_saveloop_idempotent_with_refs(user_by_api_key_in_env):
```

Comment:
i feel like having these skipped tests aren't that bad, unless they are using old fixtures or otherwise dead code

Reply:
This uses the old `weave.type()` stuff which is no longer supported